syft/__init__.py

```diff
@@ -84,6 +81,17 @@
 # import common
 import syft.common.util
 
+# Import grids
+from syft.grid.private_grid import PrivateGridNetwork
+from syft.grid.public_grid import PublicGridNetwork
+
+# Import sandbox
+from syft.sandbox import create_sandbox, make_hook
+
+# Import federate learning objects
+from syft.frameworks.torch.fl import FederatedDataset, FederatedDataLoader, BaseDataset
+from syft.federated.train_config import TrainConfig
```

Comment:
We have an outstanding TODO to vaporize `TrainConfig`, which is definitely out of scope for this PR, but I'd avoid importing it here just so we don't give the impression that people should be using it.

Reply:
Will do, the only problem is that because of global code usage in serde.py, the usage of that interface requires that all of the classes that uses the interfaces are loaded before the global code execution. I'll think of some workaround on it or maybe we want to remove the global code at the expense of some performance? (or maybe at least do some caching, like, we wait for actual execution and afterwards we store the result globally)